app/assets/stylesheets/finder_frontend.scss

```diff
@@ -354,3 +354,9 @@ mark {
   padding-left: 13px;
   padding-right: 13px;
 }
+
+.facets__content {
+  .gem-c-select__select--full-width {
+    min-width: 100%;
```

Comment:
A select component with the full_width option (which this is) already has width set to 100%, so there's something odd going on here. It looks like the select element itself is escaping from the bounds of the component itself, which I think should be investigated at the component level.

Reply:
@andysellick Yep it has `width: 100%` on it, but `.govuk-select` has `min-width: 11.5em;` which causes the issue. Should I add the `min-width: 100%` to the select component instead?

Reply:
Agree that it's probably easier to address this in a separate PR.